python/prairielearn.py

```diff
@@ -52,6 +53,78 @@ class QuestionData(TypedDict):
 class ElementTestData(QuestionData):
     test_type: Literal['correct', 'incorrect', 'invalid']
 
+def grade_question_parameterized(data: QuestionData,
+                                 question_name: str,
+                                 grade_function: Callable[[Any], Tuple[Union[bool, float], Optional[str]]],
+                                 weight: int = 1) -> None:
+    '''
+    Grade question question_name, marked correct if grade_function(student_answer) returns True in
+    its first argument. grade_function should take in a single parameter (which will be the submitted
```

Comment:
the 'in its first argument' part here is confusing. don't you mean, 'in the first element of the tuple that is returned?'